.simplecov

```diff
@@ -0,0 +1,16 @@
+require 'simplecov-json'
+
+SimpleCov.start 'rails' do
+  enable_coverage :branch
+
+  add_filter '/test'
+  add_filter '/spec'
+  add_filter 'config'
+
+  formatter SimpleCov::Formatter::MultiFormatter.new([
+    SimpleCov::Formatter::JSONFormatter,
+    SimpleCov::Formatter::HTMLFormatter
+  ])
+
+  SimpleCov.minimum_coverage 0
```

Comment:
I understand that value will change as we aim to cover as much as possible
but can we raise that value to a higher value (maybe current coverage) ?